internal/commands/root.go

```diff
@@ -205,6 +210,10 @@ func NewAstCLI(
 		chatCmd,
 	)
 
+	if wrappers.FeatureFlags[featureFlagsConstants.BYOR_ENABLED] {
```

Comment:
please set default valu + test it

Reply:
Done

Reply:
make sure that it happen (manualy), add be changes/after ben merge